### PR TITLE
[feat] Added Logout Icon on Navbar

### PR DESCRIPTION
**Pull request by rishichourasia**

### CODE REVIEW

File: components/cart.html [hunk 1]
```diff
@@ -43,6 +43,11 @@
 						<i class="far fa-shopping-cart fa-lg"></i>
 					</div>
 				</a>
+				<a href="/"
```
Comment (Chiragmodi01) [inline components/cart.html:46]: You might want to format your code a bit.

**Review by Chiragmodi01 — APPROVED** [review]
Nothing much to add. Looks fine, good to merge!

File: components/cart.html (hunk 1 above)
Comment (rishichourasia) [reply components/cart.html:46]: yeah i know , for some reason my prettier isn't working well . causes problem in only anchor tags.

**Review by AnuragSahu11 — COMMENTED** [review]
Keep up the good work

**Pull request merged**